### PR TITLE
Added readme

### PR DESCRIPTION
**Pull request by mileni67**
This is a demo version of the README. The GIF with the website will be available a bit later. If you have any comments, please write them down :)

### CODE REVIEW

File: src/Features/Navigation/Search/index.js
Comment (Siedemus) [inline src/Features/Navigation/Search/index.js:1]: we dont need this, this change is actually on main branch

File: README.md
Comment (Siedemus) [inline README.md:1]: very cool readme

**Review by Siedemus — APPROVED** [review]

File: src/Features/Navigation/Search/index.js
Comment (Siedemus) [reply src/Features/Navigation/Search/index.js:1]: please, just delete this and you can merge this branch

**Review by DominikaBuczekSzkowron — APPROVED** [review]
read me is ok,
changes of Navigation are unnecessary

**Pull request merged**